fastlane/jetpack_metadata/android/iw-IL/full_description.txt

```diff
@@ -4,7 +4,7 @@ Jetpack ל-WordPress
 
 יצירה
 
-לרעיונות הנהדרים שלך מגיעה פינה חמה באינטרנט. אפליקציית Jetpack ל-Android מאפשרת לך לבנות אתרים וליצור בלוגים על התשתית של WordPress. כדאי להשתמש בה ליצירת האתר שלך.
+לרעיונות הנהדרים שלך מגיעה פינה חמה באינטרנט. אפליקציית Jetpack ל-Android מאפשרת לך ליצור אתרים ובלוגים על תשתית של WordPress. כדאי להשתמש בה ליצירת האתר שלך.
```

Comment:
Confirmed to have been taken into account by PlayStore:
<img width="1582" alt="image" src="https://user-images.githubusercontent.com/216089/188201353-d95dcbd9-bd44-46d2-941c-aaf99219e903.png">